NetBSD legacy build fails: netbsd-meminfo.c uses int64 and missing vmtotal header

Legacy Makefile build on NetBSD fails when compiling client/netbsd-meminfo.c. The file uses non-standard int64 and references struct vmtotal without including the header that defines it.

Error output:
```
netbsd-meminfo.c:25:2: error: unknown type name ‘int64’; did you mean ‘int64_t’?
netbsd-meminfo.c:31:17: error: storage size of ‘vmdata’ isn’t known
```

Environment:
- OS: NetBSD 10.1 (amd64)
- Build: legacy Makefiles (GNU make)

Repro (simplified):
1. ./configure.server (defaults)
2. gmake -j2
3. Failure in client/netbsd-meminfo.c

Suggested fix:
- Include <sys/vmmeter.h>
- Use int64_t instead of int64

Minimal diff:
```c
#include <sys/vmmeter.h>
...
int64_t physmem;
```

--- a/client/netbsd-meminfo.c
+++ b/client/netbsd-meminfo.c
@@ -23,7 +23,7 @@ static char rcsid[] = "$Id$";
 int main(int argc, char *argv[])
 {
 	int hw_physmem[] = { CTL_HW, HW_PHYSMEM64 };
-int64_t physmem;
+	int64_t physmem;
 
 	int hw_pagesize[] = { CTL_HW, HW_PAGESIZE };
 	int pagesize;

--- a/client/netbsd-meminfo.c
+++ b/client/netbsd-meminfo.c
@@ -14,6 +14,7 @@ static char rcsid[] = "$Id$";
 #include <sys/param.h>
 #include <sys/types.h>
 #include <sys/sysctl.h>
+#include <sys/vmmeter.h>
 #include <sys/swap.h>
 #include <unistd.h>
 #include <stdlib.h>
@@ -22,7 +23,7 @@ static char rcsid[] = "$Id$";
 int main(int argc, char *argv[])
 {
 	int hw_physmem[] = { CTL_HW, HW_PHYSMEM64 };
-	int64 physmem;
+int64_t physmem;
 
 	int hw_pagesize[] = { CTL_HW, HW_PAGESIZE };
 	int pagesize;
@@ -71,4 +72,3 @@ int main(int argc, char *argv[])
 
 	return 0;
 }
-

